packages/react/src/components/Accordion/AccordionItem.js

```diff
@@ -77,12 +78,9 @@ function AccordionItem({
         className={`${prefix}--accordion__heading`}
         onClick={onClick}
         onKeyDown={onKeyDown}
-        title={iconDescription}
+        title={title}
```

Comment:
```suggestion
        title={title || iconDescription}
```

should `title` default to the `iconDescription` in the meantime value for backwards compatibility? and then we can remove the eslint rule comment above

Reply:
Makes sense to me! Thanks 👍